src/test/locale/sr-cyrl.js

```diff
@@ -170,7 +170,7 @@ test('from', function (assert) {
     );
     assert.equal(
         start.from(moment([2007, 1, 28]).add({ m: 44 }), true),
-        '44 минута',
+        '44 минуте',
```

Comment:
I don't think there is any case in our language when this would be correct. Could you back this up with something official?

Reply:
[Here](https://zelenaucionica.com/dva-minuta-ili-dve-minute/). Masculine and feminine genders for noun "minuta" are equal when it comes to numbers with last digit between 2 and 4 and the second to last digit is not 1 (2-4, 22-24, 32-34...). When last digit is more than 4, it's "minuta".

Maybe it doesn't sound correct however it is. Since the rule is loose regarding minutes, I could change it.

Reply:
Now that I think about it even more, I think "minute" is actually in Bosnian and "minuta" is in Serbian. I'll change this to "minuta".

Reply:
Well, it says that both are correct since one is masculine and the other is feminine genders of the noun. But you'll agree that "minuta" in that case sounds far more natural and common for our language and region :)